src/internal/SetupResumeAcceptor.cpp

```diff
@@ -143,8 +143,19 @@ void SetupResumeAcceptor::processFrame(
         break;
       }
 
-      removeConnection(transport);
-      onResume(std::move(transport), std::move(resumeParams));
+      if (!onResume(transport, std::move(resumeParams))) {
+        transport->outputFrameOrEnqueue(frameSerializer->serializeOut(
+            Frame_ERROR::connectionError("Resumption Failure")));
```

Comment:
Will need to be added here https://github.com/rsocket/rsocket-cpp/blob/4407a4a0cb0781da3f8217160d253e0742d2ed01/src/framing/Frame.h#L62